src/global/global.cpp

```diff
@@ -359,6 +359,10 @@ void parse_param(char *name, char *value, struct parameters *parms)
     parms->yaw = atof(value);
   } else if (strcmp(name, "polarization") == 0) {
     parms->polarization = atof(value);
+  } else if (strcmp(name, "R") == 0) {
```

Comment:
You might consider renaming R to ```R_blast``` similarly to how you've named ```P_blast``` which I think is quite descriptive. 

Although I am a proponent of the single letter variables, you can also consider replacing "R" and "P" with "radius" and "pressure"

Reply:
Ok, I've changed `R` to `radius`. I think I'm going to leave `P_blast` alone because I don't want to change `P` in every single parameter file and I want it to be clear that `P` and `P_blast` are the same thing, just with slightly different uses.